src/components/popover-menu/popover-menu-panel.tsx

```diff
@@ -1,31 +1,16 @@
 import { PopoverPanel as HeadlessUiPopoverPanel } from "@headlessui/react";
 import React from "react";
 import { usePopoverMenuContext } from "./popover-menu-context";
+import { PopoverMenuPanelGroup } from "./popover-menu-panel-group";
+import { PopoverMenuPanelItem } from "./popover-menu-panel-item";
+import { PopoverMenuPanelDivider } from "./popover-menu-panel-divider";
+import { PopoverMenuPanelTitle } from "./popover-menu-panel-title";
 
-export interface PopoverMenuPanelItemProps {
+export interface PopoverMenuPanelProps {
     children: React.ReactNode;
-    onClick?: () => void;
 }
 
-const PopoverMenuPanelItem = ({ children, onClick }: PopoverMenuPanelItemProps) => {
-    return (
-        <button
-            className="flex w-full cursor-pointer items-center overflow-hidden px-4 py-2 text-sm font-normal hover:bg-neutral-100 focus:ring-2 focus:ring-primary-200"
-            type="button"
-            tabIndex={0}
-            onClick={onClick}
-            onKeyDown={onClick}
-        >
-            {children}
-        </button>
-    );
-};
-
-export interface NavigationPopoverPanelProps {
-    children: React.ReactNode;
-}
-
-const PopoverMenuPanel = ({ children }: NavigationPopoverPanelProps) => {
+const PopoverMenuPanel = ({ children }: PopoverMenuPanelProps) => {
```

Comment:
I'm seeing these 2 linter errors in this file

![CleanShot 2024-08-06 at 10 07 07](https://github.com/user-attachments/assets/6855b278-a79f-43e7-9eec-6a41fb38b956)

I imagine it's because the props are not being exported?

I'm seeing the same for `src/components/popover-menu/popover-menu.tsx`

Reply:
ups, well, it seems that we need to export the interfaces here, regardless if we need to import them anywhere or not, but you need to export them for the bundler i guess. I’ve added the exports now ✅ 👌